lib/fourchette/pgbackups.rb

```diff
@@ -7,42 +10,42 @@ def initialize
   end
 
   def copy(from, to)
-    ensure_pgbackups_is_present(from)
-    ensure_pgbackups_is_present(to)
-
     from_url, from_name = pg_details_for(from)
     to_url, to_name = pg_details_for(to)
 
-    @client =  Heroku::Client::Pgbackups.new pgbackup_url(from) + '/api'
-    @client.create_transfer(from_url, from_name, to_url, to_name)
-  end
+    raw_attachment = @heroku.legacy_client.get_attachments(to).body[0]
+    attachment = Heroku::Helpers::HerokuPostgresql::Attachment.new raw_attachment
```

Comment:
Getting the 0 indexed attachment does not work for me. We should probably analyze each attachment until we find the attachment with a name starting with ```HEROKU_POSTGRESQL```